src/components/Mentees/Mentee.js

```diff
@@ -0,0 +1,518 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { withStyles } from "@material-ui/core/styles";
+import FormHelperText from "@material-ui/core/FormHelperText";
+import Typography from "@material-ui/core/Typography";
+import FormControl from "@material-ui/core/FormControl";
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+import { Link } from "react-router-dom";
+import Snackbar from "@material-ui/core/Snackbar";
+import SnackbarContentWrapper from "../SnackbarContentComponent/SnackbarContentComponent";
+import { Redirect } from "react-router-dom";
+import * as R from "ramda";
+import { editMentee, deleteMentee } from "../../firebase/operations";
+import { adminApp } from "../../firebase/firebase.js";
```

Comment:
We're not using `adminApp`, we can delete it.

Reply:
Done